example.py

```diff
@@ -44,6 +44,9 @@ def simulate(car, time_step=0.0002, interval=1):
 
 if __name__ == "__main__":
     car = Car()
+    # road = Road(length= 6, mode="triagle", frequency=1.8, amplitude= 0.1)
```

Comment:
This needs x_min like below

```python
    road = Road(length= 6, x_min=-3, mode="triagle", frequency=1.8, amplitude= 0.1)
    car = Car(road_func=road)
```

Reply:
Please see comment on PR #4 (https://github.com/nrsyed/half-car/pull/4#discussion_r310421270) on default and hard-coded `Road` `length` and `x_min` values.

Reply:
I closed #4 in favour of this #5 as it is needed here to make this PR work.